Assets/Scripts/Character/CharacterController.cs

```diff
@@ -0,0 +1,121 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using MyBox;
+
+public class CharacterController : MonoBehaviour {
```

Comment:
This should be renamed because `CharacterController` is an existing Unity component name